src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String toString() {
+        return getStatusIcon() + getDescription();
+    }
+
+    public String getStatusIcon() {
+        return "[" + (isDone ? "X" : " ") + "] ";
```

Comment:
Perhaps the `isDone` ternary operation can be extracted into a new line to simplify the return expression.